samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
> Why?

Because it is a non-standard-compliant file 😀. I would not want someone to think that it represents a correct ABNF grammar.

However, I just removed `tranquil.abnf`. I hope that is a compromise that works for everyone.